### PR TITLE
NMS-14301: Configure port for juniper devices

### PR DESCRIPTION
**Pull request by cgorantla**
Add missing docs

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14301

### CODE REVIEW

**Review by christianpape — APPROVED** [review]
LGTM.

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 1]
```diff
@@ -49,6 +52,36 @@ include::debian-ubuntu/firewall-core.adoc[]
 --
 ====
 
+== Configure TFTP IPAddress
+
+The device config backup feature allows the retrieval of network device configurations by using the TFTP
+protocol. To achieve this, a TFTP server needs to be started on the OpenNMS instance or on a minion. Normally
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/configuration.adoc:58]:
```suggestion

To achieve this, a TFTP server needs to be started on the OpenNMS instance or on a Minion. 
```
Separate lines

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 2]
```diff
@@ -105,14 +138,15 @@ Additional Parameters
 * `eventReason` - If the backup failed, contains a short description of the issue.
 
 
-== Retrieve device configuration on demand
+== Backup device configuration on demand
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/configuration.adoc:141]:
```suggestion
== Back up device configuration on demand
```

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 3]
```diff
@@ -105,14 +138,15 @@ Additional Parameters
...
 
-You can use the Rest API to trigger device configuration retrieval outside of a monitor schedule.
+You can manually backup devices from OpenNMS UI under Info>Device Configs.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/configuration.adoc:143]:
```suggestion
You can manually back up devices from the OpenNMS UI under *Info>Device Configs*.
```

File: docs/modules/operation/pages/device-config-backup/configuration.adoc [hunk 4]
```diff
@@ -105,14 +138,15 @@ Additional Parameters
...
+You can also use the Rest API to trigger device configuration retrieval outside of a monitor schedule.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/configuration.adoc:144]:
```suggestion
You can also use the REST API to trigger device configuration retrieval outside of a monitor schedule.
```

File: docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc [hunk 5]
```diff
@@ -18,12 +18,13 @@ For example, `0 0 0 * * ?` runs every 24 hrs at 12:00 a.m. `0 0 1 * * ?` runs ev
...
 . Click on the edit icon and click *Add Node*.
 . Set up your requisition as described in xref:operation:provisioning/getting-started.adoc#requisition-create[Create a requisition].
+. Add `DeviceConfig` service on the interface where we are backing up the config. If device has multiple config types, you can add services with prefix `DeviceConfig-`.
 . Click *Add Meta-Data*, and define the following key-value pairs:
 .. `dcb:username`: username for the device
 .. `dcb:password`: password for the device
 .. `dcb:script-file`: name of the device configuration script for this type of device without .dcb extension
 .. `dcb:schedule`: a cron expression to specify the time to fetch the device backup configuration (default is `never`)
-.. `retention-period`: an optional Java `Period` expression to specify the length of time before "stale" configuration data is deleted from the database (default is `P1Y`, one year)
+.. `dcb:retention-period`: an optional Java `Period` expression to specify the length of time before "stale" configuration data is deleted from the database (default is `P1Y`, one year)
```
Comment (Bonrob2) [inline docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc:27]:
```suggestion
.. `dcb:retention-period`: an optional Java period expression to specify the length of time before "stale" configuration data is deleted from the database (default is `P1Y`, one year).
```

File: docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc [hunk 6]
```diff
@@ -61,10 +61,12 @@ Overwrite in metadata as `dcb:ssh-timeout`.
 | schedule
 | Schedule to fetch configuration, in the form of cron expression.
   Defaults to never run; overwrite this in metadata with proper cron expression.
+  Overwrite in metadata as `dcb:schedule`
```
Comment (Bonrob2) [inline docs/modules/reference/pages/service-assurance/monitors/DeviceConfigMonitor.adoc:64]:
```suggestion
  Overwrite in metadata as `dcb:schedule`.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
a few changes to the docs. thanks!

**Comment by cgorantla** [comment]
Thanks @Bonrob2 for the review. Updated!

**Review by Bonrob2 — APPROVED** [review]
docs LGTM!

**Pull request merged**